crates/cheatcodes/src/evm.rs

```diff
@@ -108,7 +108,7 @@ impl Cheatcode for loadCall {
                 // If storage slot is untouched and load from a target that copies storage from
                 // a source address with arbitrary storage, then copy existing arbitrary value.
                 // If no arbitrary value generated yet, then the random one is saved and set.
-                let rand_value = ccx.state.rng().gen();
+                let rand_value = ccx.state.test_runner().rng().gen();
```

Comment:
```suggestion
                let rand_value = ccx.state.rng().gen();
```

Reply:
oops, missed suggestion but was already included in https://github.com/foundry-rs/foundry/pull/8882/commits/0a56dfcf2110f55c08bf5fdc86330970b57c07dc